domain/src/Commands/DomainCommands.php

```diff
@@ -0,0 +1,1108 @@
+<?php
+
+namespace Drupal\domain\Commands;
+
+use Consolidation\OutputFormatters\StructuredData\PropertyList;
+use Drupal\Component\Plugin\Exception\PluginException;
+use Drupal\Core\Config\StorageException;
+use Drush\Commands\DrushCommands;
+use Drupal\Component\Utility\Html;
+use Drupal\domain\DomainInterface;
+use Drupal\Component\Serialization\Yaml;
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityStorageException;
+use Drupal\Component\Plugin\Exception\PluginNotFoundException;
+use Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException;
+
+
+/**
+ * A Drush commandfile.
+ *
+ * In addition to this file, you need a drush.services.yml
+ * in root of your module, and a composer.json file that provides the name
+ * of the services file to use.
+ *
+ * See these files for an example of injecting Drupal services:
+ *   - http://cgit.drupalcode.org/devel/tree/src/Commands/DevelCommands.php
+ *   - http://cgit.drupalcode.org/devel/tree/drush.services.yml
+ */
+class DomainCommands extends DrushCommands {
+
+  /** @var \Drupal\domain\DomainStorageInterface $domain_storage */
+  protected $domain_storage = NULL;
+
+  /** @var string[] Array of special-case policies for reassigning content. */
+  protected $reassignment_policies = ['prompt', 'default', 'ignore']; // + machine name;
+
+  /**
+   * Get a domain storage object or throw an exception.
+   *
+   * @return \Drupal\domain\DomainStorageInterface
+   *
+   * @throws \Drupal\domain\Commands\DomainCommandException
+   */
+  protected function getStorage() {
+    if ($this->domain_storage) {
```

Comment:
Probably more consistent to use `!is_null` here.

Reply:
Done.